static/js/redux/ui/top_bar_advising.jsx

```diff
@@ -76,6 +79,27 @@ class TopBarAdvising extends React.Component {
     }
 
     render() {
+        const SISImportDataModalButton = (
+            <div className="cal-btn-wrapper">
+                <button
+                    onClick={() => this.props.triggerSISImportDataModal()}
+                    className="save-timetable add-button"
+                    data-tip
+                    data-for="mock-btn-tooltip"
+                >
+                    <img src="/static/img/star.png" alt="STAR" style={{ marginTop: '2px' }} />
+                </button>
+                <ReactTooltip
+                    id="mock-btn-tooltip"
+                    class="tooltip"
+                    type="dark"
+                    place="bottom"
+                    effect="solid"
+                >
+                    <span>SIS Import Data Modal</span>
+                </ReactTooltip>
+            </div>
+        );
```

Comment:
I think this is okay for now but add a TODO comment to remove this and instead add an "Import Data" button.  Then, create an Issue to track this TODO. Maybe this issue can be called "Replace star with an Import Data button that triggers the Import Data modal"

Reply:
Will change button

Reply:
I noticed that Semesterly makes use of Fontawesome.com icons if you want to look into that, although I think just a button that says "import data" would be fine